### PR TITLE
Added description for throttle in RxSwift

### PR DESCRIPTION
**Pull request by 1397v**
Also added the difference between debounce and throttle in the RxSwift framework

### CODE REVIEW

File: documentation/operators/debounce.html [hunk 1]
```diff
@@ -315,9 +315,41 @@ <h4>Sample Code</h4>
...
+    <div class="code swift">
+    	<pre>
+    		-----1---2-34-5----6---->
+    		debounce(2)
+    		--------1--------5----6->
+    	</pre>
+    </div>
+    <ul>
+    	<li>
+    		<code>throttle</code>. Calls the specified function when the specified period has passed, regardless of the number of events that have occurred during that time.
```
Comment (DavidMGross) [inline documentation/operators/debounce.html]: In that case, you should probably move the discussion of the "throttle" operator in RxSwift from this page to http://reactivex.io/documentation/operators/sample.html as it behaves more like sample or throttleLast in other implementations.

Comment (1397v) [reply documentation/operators/debounce.html]: Hmmm, I think its behaviour is more like `debounce`, indeed, its mention in that (page)[http://reactivex.io/documentation/operators/debounce.html]. Changing its location would make the other `throttle` operators in that (page)[http://reactivex.io/documentation/operators/debounce.html] to be misplaced.

Comment (DavidMGross) [reply documentation/operators/debounce.html]: There are two throttle operators that are covered on the "sample" page -- throttleFirst and throttleLast. From the description, the RxSwift version of "throttle" looks like it behaves more like one of those than it does the debounce or throttleWithTimeout operators in other languages.

I think RxPHP's "throttle" probably also would be better placed on the "sample" page than on the "debounce" page, based on its description.

Comment (1397v) [reply documentation/operators/debounce.html]: ok

**Pull request closed without merging**